torchreid/engine/image/am_softmax.py

```diff
@@ -262,10 +263,14 @@ def forward_backward(self, data):
 
                 total_loss += coeff_mutual_learning * mutual_loss / float(num_mutual_losses)
 
-
             total_loss.backward(retain_graph=self.enable_metric_losses)
 
             for model_name in model_names:
+                # TODO(kshpv, lbeynens): Change this ugly code. The bug appears trying to
```

Comment:
this TODO  has been here for a long time

Reply:
I've changed it, please check

Reply:
I mean, if somebody is going to actually do this TODO? =)